docs/how-to/h-deploy-aks.md

```diff
@@ -0,0 +1,235 @@
+# How to deploy on AKS
+
+[Azure Kubernetes Service](https://learn.microsoft.com/en-us/azure/aks/) (AKS) allows you to quickly deploy a production ready Kubernetes cluster in Azure. To access the AKS Web interface, go to [https://portal.azure.com/](https://portal.azure.com/).
+
+## Summary
+* [Install Azure and Juju tooling](#install-client-environment)
+* [Create AKS Cluster](#create-aks-cluster)
+* [Bootstrap Juju controller on AKS](#bootstrap-juju-controller-on-aks)
+* [Deploy charms](#deploy-charms)
+* [Display deployment information](#display-deployment-information)
+* [Clean up](#clean-up)
+
+---
+
+## Install Client Environment
+
+### Juju 
+
+Install Juju via snap:
+
+```shell
+sudo snap install juju
+```
+
+Check that the Juju version is correctly installed:
+
+```shell
+juju version
+```
+
+which should show:
+
+```shell
+3.5.2-genericlinux-amd64
+```
+
+### Azure CLI
+
+Follow the [user guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli-linux?pivots=apt) for installing the Azure CLI on Linux distributions.
+
+Verify that it is correctly installed running the command below:
+
+```shell
+az --version
+```
+
+which should show the following output:
+
+```shell
+azure-cli                         2.65.0
+
+core                              2.65.0
+telemetry                          1.1.0
+
+Dependencies:
+msal                              1.31.0
+azure-mgmt-resource               23.1.1
+
+Python location '/opt/az/bin/python3'
+Extensions directory '/home/deusebio/.azure/cliextensions'
+
+Python (Linux) 3.11.8 (main, Sep 25 2024, 11:33:44) [GCC 11.4.0]
+
+Legal docs and information: aka.ms/AzureCliLegal
+
+
+Your CLI is up-to-date.
+```
+
+## Create AKS cluster
+
+Login to your Azure account:
+```shell
+az login
+```
+
+This following examples in this guide will use the single server AKS in location `eastus` - feel free to change this for your own deployment.
+
+Create a new [Azure Resource Group](https://learn.microsoft.com/en-us/cli/azure/manage-azure-groups-azure-cli):
+
+```shell
+az group create --name aks --location eastus
+```
+
+Export the deployment name for further use:
+```shell
+export JUJU_NAME=aks-$USER-$RANDOM
```

Comment:
give some explanation of this string

Reply:
Uhm, being this an how-to, it is probably not need to provide a specific name for Juju, and I will just leave it up to the user to pick one

Reply:
I have actually renamed this variable, and not used the environemtn, but a placeholder, as done for other "variables"